be/src/olap/task/engine_batch_load_task.cpp

```diff
@@ -344,35 +307,26 @@ Status EngineBatchLoadTask::_delete_data(const TPushReq& request,
     Status res = Status::OK();
 
     if (tablet_info_vec == nullptr) {
-        LOG(WARNING) << "invalid tablet info parameter which is nullptr pointer.";
-        return Status::OLAPInternalError(OLAP_ERR_CE_CMD_PARAMS_ERROR);
+        return Status::InvalidArgument("invalid tablet_info_vec which is nullptr");
     }
 
     // 1. Get all tablets with same tablet_id
     TabletSharedPtr tablet =
             StorageEngine::instance()->tablet_manager()->get_tablet(request.tablet_id);
     if (tablet == nullptr) {
-        LOG(WARNING) << "can't find tablet. tablet=" << request.tablet_id;
-        return Status::OLAPInternalError(OLAP_ERR_TABLE_NOT_FOUND);
+        return Status::InternalError("could not find tablet {}", request.tablet_id);
     }
 
     // 2. Process delete data by push interface
     PushHandler push_handler;
-    if (request.__isset.transaction_id) {
-        res = push_handler.process_streaming_ingestion(tablet, request, PUSH_FOR_DELETE,
-                                                       tablet_info_vec);
-    } else {
-        res = Status::OLAPInternalError(OLAP_ERR_PUSH_BATCH_PROCESS_REMOVED);
+    if (!request.__isset.transaction_id) {
+        return Status::InvalidArgument("transaction_id is not set");
     }
-
+    res = push_handler.process_streaming_ingestion(tablet, request, PUSH_FOR_DELETE,
+                                                   tablet_info_vec);
     if (!res.ok()) {
-        LOG(WARNING) << "fail to push empty version for delete data. "
-                     << "res=" << res << "tablet=" << tablet->full_name();
         DorisMetrics::instance()->delete_requests_failed->increment(1);
-        return res;
     }
-
-    LOG(INFO) << "finish to process delete data. res=" << res;
     return res;
```

Comment:
All these tasks have a same problem. If the task is success, there is no log. So if a table lost some data or data is wrong, we could not find what happened on this tablet. I think we should print some success log to help us debug. Maybe could add a log in EngineTask.

Reply:
Such log will be print in `_push_worker_thread_callback`:
```c++
        if (status.ok()) {
            LOG_INFO("successfully execute push task")
                    .tag("signature", agent_task_req.signature)
                    .tag("tablet_id", push_req.tablet_id)
                    .tag("push_type", push_req.push_type);
            ++_s_report_version;
            finish_task_request.__set_finish_tablet_infos(tablet_infos);
        } else {
            LOG_WARNING("failed to execute push task")
                    .tag("signature", agent_task_req.signature)
                    .tag("tablet_id", push_req.tablet_id)
                    .tag("push_type", push_req.push_type)
                    .error(status);
        }
```